tests/PropertyList.py

```diff
@@ -102,14 +102,14 @@ def testExists(self):
 
     def testGetVector(self):
         apl = dafBase.PropertyList()
-        v = (42, 2008, 1)
+        v = [42, 2008, 1]
         apl.setInt("ints", v)
-        apl.setInt("ints2", (10, 9, 8))
+        apl.setInt("ints2", [10, 9, 8])
```

Comment:
Can't `setInt` take a tuple? That's an API change isn't it? How often does our code use a tuple rather than a list here?